src/ApplicationLogs/LogReader.cs

```diff
@@ -25,7 +25,7 @@ namespace Neo.Plugins
 {
     public class LogReader : Plugin
     {
-        private DB db;
+        private DB _db;
```

Comment:
Changes like this make it much harder to review PRs. Reviewers need to check each change to see if it is substantive or name change.

Reply:
This pr contains no substantive change, its just typos, comments, name styles. I open this pr simply because i found some  typos when i was going to work on  another feature.